src/databricks/labs/ucx/source_code/linters/python_ast.py

```diff
@@ -130,28 +142,35 @@ def _get_attribute_value(cls, node: Attribute):
             logger.debug(f"Missing handler for {name}")
         return None
 
-    def infer_values(self) -> Iterable[InferredValue]:
+    def infer_values(self, state: CurrentSessionState | None = None) -> Iterable[InferredValue]:
+        if state is not None:
```

Comment:
check for the presence of named parameters here

Reply:
done